lib/walletInfo.ts

```diff
@@ -0,0 +1,91 @@
+import { Platform } from "react-native";
+import { SUITE_NAME } from "~/lib/constants";
+
+let UserDefaults: any;
+let SharedPreferences: any;
+
+if (Platform.OS === "ios") {
```

Comment:
Why do we use different storage and cannot use the standard expo storage?

Reply:
if it is required, it could be conditionally enabled based on an environment variable, so it doesn't break expo go by default